docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -127,17 +150,72 @@ PAF-of-1 Cause/Risk Pairs
-++++++++++++++++++++++
+Stratifying Exposure Variables by Baseline Intervention Coverage
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+From GBD we obtain overall population estimates for our outcomes of interest
+(prevalence of vitamin A deficiency, birth prevalence of neural tube defects,
+distribution of haemoglobin levels, and distribution of birth weights). Because
+there exists baseline coverage of our interventions --- i.e. there is already
+some food fortification occurring in the locations we are simulating --- these
+overall population estimates include both people who are eating fortified food
+and those who are not.
+
+When we introduce additional food fortification into the population, only those
+who are *not* already eating fortified food should receive the benefit.
+Therefore, in order to properly measure the effects of our interventions, we
+must first stratify our population into those who are already covered vs. not
+covered by the forticant of interest. We then need to calculate the risk
+exposure (or other relevant outcome) in each coverage stratum to get a complete
+picture of the baseline population. For example, we will need to know the
+prevalence of vitamin A deficiency or the distribution of haemoglobin in the
+subpopulation already eating fortified food as well as in the subpopulation not
+eating fortified food.
+
+These coverage-stratified exposures are calculated so that:
+
+1.  Their population-weighted average matches the overall population estimate
+    from GBD, and
+2.  The differential exposure between the subpopulations matches the
+    corresponding effect size for our intervention.
+
+For example, the ratio of the prevalence of vitamin A deficiency between the
+fortified and unfortified groups should match the risk ratio of vitamin A
+fortification on vitamin A deficiency from the literature, and the difference in
+mean haemoglobin level between the fortified and unfortified groups should match
+the mean shift in haemoglobin from iron fortification found in the literature.
+When we stratify the population in this way, we are assuming that the effect
+sizes of fortification found in the literature are generalizable to the
+populations we are simulating (note that if this assumption is *not* true, we
+should not be simulating the interventions in theses populations at all).
+
+.. todo::
 
-Effect size stratification for baseline population
---------------------------------------------------
+  Come up with some examples to show what would happen if we *didn't* first
+  stratify by existing coverage. I.e., why is this baseline stratification
+  important? It is not totally obvious to me what the effect would be if we
+  didn't do it (e.g. would we end up overestimating or underestimating the true
+  effect?) -- I think the net effect depends on the level of existing coverage,
+  the shape of the underlying exposure distribution, and the magnitude of the
+  effect size.
```

Comment:
oh!! I have an example I've written out for myself for this. I can add it once this is merged as a new branch?

Reply:
Cool!